### PR TITLE
[plugin.video.infowars] 2.0.8

### PR DESCRIPTION
**Pull request by Spinalcracker**
Version 2.0.8
- Removed Censored Content (Spinalcracker)
- Cleaned up Menu System
- Added New Shows
- Added Millie Weaver Youtube Channel

*** Removed old bad code as per previous pull request

### CODE REVIEW

File: plugin.video.infowars/default.py [hunk 1]
```diff
@@ -331,15 +338,18 @@ def playYoutube(url):
...
+    IW_addon.add_directory({'mode': 'PlayURL','url':'https://streaminfo-live.secure.footprint.net/hls-live/streamroot2-streaminfo2/_definst_/live.m3u8'},{'title':  cFL_('War Room with Owen Shroyer - (Loops After Airing)','purple')},is_folder=False,img=WarRoomIcon,fanart=WarRoomFanart)
+    IW_addon.add_directory({'mode': 'PlayURL','url':'https://infowarslive-lh.akamaihd.net/i/MikeAdams_1@704916/master.m3u8'},{'title':  cFL_('Counter Think with Mike Adams - (Loops After Airing)','orange')},is_folder=False,img=CTIcon,fanart=CTFanart)
+    IW_addon.add_directory({'mode': 'PlayURL','url':'https://infowarslive-lh.akamaihd.net/i/infowarsevent_1@366809/master.m3u8'},{'title':  cFL_('Live Shows & Special Events','yellow')},is_folder=False,img=IWLiveSEIcon,fanart=IWLiveSEFanart)
+	#IW_addon.add_directory({'mode': 'ClipsSubMenu','title':'Infowars Nightly News'},{'title':  cFL_('Infowars Clips','red')},is_folder=True,img=IW_artIcon,fanart=IW_artFanart)
```
Comment (Rechi) [inline plugin.video.infowars/default.py]: Why not remove all this commented out shows and look it up in the git history if needed later?

File: plugin.video.infowars/default.py [hunk 2]
```diff
@@ -354,63 +364,11 @@ def Menu_MainMenu(): #The Main Menu
     li_url = IW_addon.build_plugin_url(queries)
     #xbmcplugin.addDirectoryItem(int(sys.argv[1]), li_url, listitem,isFolder=True)
     #IW_addon.add_directory({'mode': 'NightlyNewsSubMenu','title':'Infowars Nightly News'},{'title':  cFL_('Infowars Nightly News','red')},is_folder=True,img=IW_artIcon,fanart=IW_artFanart)
-    IW_addon.add_directory({'mode': 'ClipsSubMenu','title':'Infowars Clips'},{'title':  cFL_('Infowars Clips','yellow')},is_folder=True,img=IW_artIcon,fanart=IW_artFanart)
+    #IW_addon.add_directory({'mode': 'ClipsSubMenu','title':'Infowars Clips'},{'title':  cFL_('Infowars Clips','yellow')},is_folder=True,img=IW_artIcon,fanart=IW_artFanart)
```
Comment (Rechi) [inline plugin.video.infowars/default.py]: same

File: plugin.video.infowars/default.py [hunk 3]
```diff
@@ -354,63 +364,11 @@ def Menu_MainMenu(): #The Main Menu
...
-                url = "plugin://plugin.video.youtube/play/?video_id=%s" % video_id
-                #IW_addon.log('SPINAL VIDEO_ID IS %s' % video_id)
-                PlayURL(url)
-                break
-    eod()
-
-def Real_News_WDK_Sub_Menu(title=''): #The Main Menu
-    WhereAmI('@ Real News Live')
+    #IW_addon.add_directory({'mode': 'HistoricShowsSubMenu','title':'Past Alex Jones Shows(video)'},{'title':  cFL_('Past Alex Jones Shows (Video)','yellow')},is_folder=True,img=AJSIcon,fanart=IW_artFanart)
+    #IW_addon.add_directory({'mode': 'HistoricShowsAudioSubMenu','title':'Past Alex Jones Shows(video)'},{'title':  cFL_('Past Alex Jones (Audio)','yellow')},is_folder=True,img=AJSIcon,fanart=IW_artFanart)
```
Comment (Rechi) [inline plugin.video.infowars/default.py]: same

File: plugin.video.infowars/default.py [hunk 4]
```diff
@@ -805,14 +693,15 @@ def check_mode(mode=''):
...
     elif (mode=='playYoutube'): 							playYoutube('url')
     elif (mode=='DocSubMenu'): 						Documentary_Sub_Menu(_param['title'], movie_num) ## Play Video
-    elif (mode=='ClipsSubMenu'): 						Clips_Sub_Menu(_param['title']) ## Play Video
-    elif (mode=='NightlyNewsSubMenu'): 						Nightly_News_Sub_Menu(_param['title'], dialog) ## Play Video
-    elif (mode=='HistoricShowsSubMenu'): 						Historic_Shows_Sub_Menu(_param['title']) ## Play Video
-    elif (mode=='HistoricShowsAudioSubMenu'): 						Historic_Shows_Audio_Sub_Menu(_param['title']) ## Play Video
+    #elif (mode=='ClipsSubMenu'): 						Clips_Sub_Menu(_param['title']) ## Play Video
+    #elif (mode=='NightlyNewsSubMenu'): 						Nightly_News_Sub_Menu(_param['title'], dialog) ## Play Video
+    #elif (mode=='HistoricShowsSubMenu'): 						Historic_Shows_Sub_Menu(_param['title']) ## Play Video
+    #elif (mode=='HistoricShowsAudioSubMenu'): 						Historic_Shows_Audio_Sub_Menu(_param['title']) ## Play Video
```
Comment (Rechi) [inline plugin.video.infowars/default.py]: same

File: plugin.video.infowars/default.py [hunk 5]
```diff
@@ -805,14 +693,15 @@ def check_mode(mode=''):
...
     elif (mode=='PaulJosephWatsonSubMenu'): 						Paul_Joseph_Watson_Sub_Menu(_param['title']) ## Play Video
-    elif (mode=='InfoWarsLiveSubMenu'): 						Info_Wars_Live_Sub_Menu(_param['title']) ## Play Video
-    elif (mode=='RealNewsWDKLiveSubMenu'): 						Real_News_WDK_Sub_Menu(_param['title']) ## Play Video
-    elif (mode=='WarRoomLiveSubMenu'): 						War_Room_Sub_Menu(_param['title']) ## Play Video
+    elif (mode=='MillieWeaverSubMenu'): 						Millie_Weaver_Sub_Menu(_param['title']) ## Play Video
+    #elif (mode=='InfoWarsLiveSubMenu'): 						Info_Wars_Live_Sub_Menu(_param['title']) ## Play Video
+    #elif (mode=='RealNewsWDKLiveSubMenu'): 						Real_News_WDK_Sub_Menu(_param['title']) ## Play Video
+    #elif (mode=='WarRoomLiveSubMenu'): 						War_Room_Sub_Menu(_param['title']) ## Play Video
```
Comment (Rechi) [inline plugin.video.infowars/default.py]: same

**Comment by Spinalcracker** [comment]
Github webportal is creating syntax errors (tabs instead of spaces).  It is also creating additional commits.   Closing and will maybe repull later.

**Pull request closed without merging**